Cargo.toml

```diff
@@ -25,7 +25,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 serde_yaml = "0.8"
 shellexpand = "2.1"
-shrub-rs = "0.5.1"
+shrub-rs = { path = "../shrub-rs-0.5.1" }
```

Comment:
I needed to make some very small changes here as well, but I did not yet receive access to this repo via MANA. Will follow up with DAG.